test/Yoti.Auth.Tests/AttributeConverterTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Google.Protobuf;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Yoti.Auth.Tests
+{
+    [TestClass]
+    public class AttributeConverterTests
+    {
+        private const string StringValue = "{}";
+        private readonly ByteString _byteValue = ByteString.CopyFromUtf8(StringValue);
+
+        [TestMethod]
+        public void FailureInAttributeParsing_ShouldNotStopOtherAttributes()
+        {
+            var attribute1 = CreateProtobufJsonAttribute(Constants.UserProfile.StructuredPostalAddressAttribute, _byteValue);
+            var invalidAttribute = CreateProtobufJsonAttribute(Constants.UserProfile.PostalAddressAttribute, ByteString.CopyFromUtf8("invalid"));
+            var attribute2 = CreateProtobufJsonAttribute(Constants.UserProfile.PhoneNumberAttribute, _byteValue);
```

Comment:
Could we use descriptive attribute names here, something like `valid 1`, `valid 2` and `invalid`. It's fine as is, but might make the test easier to follow?

Reply:
Good point. I'm actually editing this file in another PR, so will change it in that 👍